zk-sdk/src/sigma_proofs/percentage_with_cap.rs

```diff
@@ -286,14 +291,14 @@ impl PercentageWithCapProof {
         };
 
         // generate equality proof
-        let x = Scalar::from(delta_amount);
+        let mut x = Scalar::from(delta_amount);
```

Comment:
I can't comment higher up, but should `z_max_proof` also get zeroized on line 277?

Reply:
Yeah let me know if I am missing anything, but the variable `z_max_proof` is also part of the `PercentageMaxProof` that is returned by the function so this is public value.